docs/jade-getting-started.md

```diff
@@ -1,20 +1,9 @@
 # Getting Started
 
-Ensure that you have access to the required team resources beforehand. If you
-encounter a permission error, it is likely because you are missing appropriate
-access. Join the `#github` Slack channel, click the lightning bolt in the
-channel header, and select `Join DataBiosphere`. Ask for access to Google Groups
-including `dsde-engineering`.
-
 These instructions assume you use MacOS, and that you are on the internal Broad
 network or the VPN. If the VPN is not installed, follow the instructions
 [at this link](https://broad.io/vpn).
 
-> During this process, you will need your GitHub and Docker Hub username,
-password, and personal access token for multiple steps, so make sure to have
-those handy. If you don't have those yet, see the section below, otherwise you
-can skip to [Connect Accounts](#2-connect-accounts)
-
```

Comment:
What is the thought behind moving this section?

Reply:
No good reason - I've moved it back!

Reply:
No strong concern either way, I just wanted to see if you had any thoughts on this. 👍